fe/fe-core/src/main/java/com/starrocks/credential/tencent/TencentCloudCredential.java

```diff
@@ -0,0 +1,76 @@
+// Copyright 2021-present StarRocks, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.starrocks.credential.tencent;
+
+import com.staros.proto.FileStoreInfo;
+import com.starrocks.credential.CloudConfigurationConstants;
+import com.starrocks.credential.CloudCredential;
+import org.apache.hadoop.conf.Configuration;
+
+import java.util.Map;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+public class TencentCloudCredential implements CloudCredential {
+    private final String accessKey;
+    private final String secretKey;
+    private final String endpoint;
+
+    public TencentCloudCredential(String accessKey, String secretKey, String endpoint) {
+        checkNotNull(accessKey);
+        checkNotNull(secretKey);
+        checkNotNull(endpoint);
+        this.accessKey = accessKey;
+        this.secretKey = secretKey;
+        this.endpoint = endpoint;
+    }
+
+    @Override
+    public void applyToConfiguration(Configuration configuration) {
+        configuration.set("fs.cosn.impl", "org.apache.hadoop.fs.CosFileSystem");
```

Comment:
specific reason to use CosNFileSystem to access cos? will aws sdk work or not?

Reply:
it test worked. FE using CosNFileSystem to access cos/ BE using was SDK to access cos

Reply:
FE contains aws java sdk, if it works, maybe we can use aws java sdk to the same work, unless there are some advantages that CosFileSystem provides? b.t.w., this configuration will not work unless the jar implemented CosFileSystem is packed into FE as well which makes FE fatter and fatter.

Reply:
I will update related dependencies later.

> FE contains aws java sdk, if it works, maybe we can use aws java sdk to the same work, unless there are some advantages that CosFileSystem provides? b.t.w., this configuration will not work unless the jar implemented CosFileSystem is packed into FE as well which makes FE fatter and fatter.

Reply:
If [this](https://cloud.tencent.com/document/product/436/56971) is enabled, using s3 sdk in FE would not work appropriately.